bqskit/compiler/passes/util/inject.py

```diff
@@ -0,0 +1,40 @@
+"""This module implements the InjectDataPass class."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from bqskit.compiler.basepass import BasePass
+from bqskit.ir.circuit import Circuit
+
+
+_logger = logging.getLogger(__name__)
+
+
+class InjectDataPass(BasePass):
```

Comment:
Is the idea behind this pass that people would add this part way through? Also perhaps this could be named `UpdateDataPass` or something?

Reply:
Yeah, basically there to give users more control over the execution of the passes. I like that name much better, will update shortly.